integration/sql/iface-java/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.openlineage.sql</groupId>
+  <artifactId>openlineage-sql-java</artifactId>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
hardcoded version and url pointing to example.com.

Reply:
Removed Maven